back/src/pods/storage/api/storage.api.ts

```diff
@@ -0,0 +1,103 @@
+import {
+  UserSessionContext,
+  UserSession,
+  ConnectSessionInfo,
+  VotesFromRooms,
+} from 'dals/user';
+import { roomApi } from 'pods/room';
+
+//TODO implement queries
+
+// This is just a demo approach, storing in memory session Info
+// Another way to identify users: https://stackoverflow.com/questions/6979992/how-to-get-session-id-of-socket-io-client-in-client
```

Comment:
About other ways to identify users, just went through the stackoverflow question and it was a bit strange (some posts are deprecated), if you could ellaborate here a litte bit more and add more info about another way that could be potentially better that could be great (we could even refactor this).

Reply:
Yes, and shouldn't be here either because is about Sockets and this is the mongo storage, also I would like to remove the socket from the app.ts and do as you said above with mongo.

different files for socket calls and mongo connection.

Reply:
And also with the comment we should add that in the documentation at the end, like interested links in my opinion, or in the socket file which we should create